### PR TITLE
Fix the opentelekomcloud job

### PR DESCRIPTION
**Pull request by liu-sheng**
- Add the OS_VPC_ID environment variable
- Change the OS_FLAVOR_ID and OS_FLAVOR_ID_RESIZE value
- Enable all the tests running
- Use an Ubuntu image than cirros image for testing 
- Add OS_AVAILABILITY_ZONE variable

For #63

### CODE REVIEW

**Comment by liu-sheng** [comment]
FYI, the testing pr is: https://github.com/liu-sheng/terraform-provider-opentelekomcloud/pull/1

File: playbooks/terraform-provider-opentelekomcloud-acceptance-test-opentelekomcloud/run.yaml [hunk 1]
```diff
@@ -40,12 +39,19 @@
...
+          fi
           export OS_SWIFT_ENVIRONMENT=1
+
           # Run except the DNS/FW/LB test 100 testcases at a time
-          testcases=`go test github.com/gator1/terraform-provider-opentelekomcloud/opentelekomcloud/ -v -list 'Acc'`
+          testcases=`go test ./opentelekomcloud/ -v -list 'Acc'`
           testcases=`echo "$testcases" | sed '$d' | grep -v -e FW -e LB`
-          # echo "$testcases" | xargs -t -n100 sh -c 'TF_ACC=1 go test github.com/gator1/terraform-provider-opentelekomcloud/opentelekomcloud/ -v -timeout 120m -run $(echo "$@" | tr " " "|")' argv0 2>&1 | tee $TEST_RESULTS_TXT
+          echo "$testcases" | xargs -t -n100 sh -c 'TF_ACC=1 go test ./opentelekomcloud/ -v -timeout 120m -run $(echo "$@" | tr " " "|")' argv0 2>&1 | tee $TEST_RESULTS_TXT
+          # TF_ACC=1 go test ./opentelekomcloud/ -v -run TestAccNetworkingV2Subnet_basic 2>&1 | tee $TEST_RESULTS_TXT
```
Comment (animationzl) [inline playbooks/terraform-provider-opentelekomcloud-acceptance-test-opentelekomcloud/run.yaml]: There is no need to keep this line in code.

Comment (animationzl) [inline playbooks/terraform-provider-opentelekomcloud-acceptance-test-opentelekomcloud/run.yaml]: There is no need to keep this line in code.

Comment (liu-sheng) [reply playbooks/terraform-provider-opentelekomcloud-acceptance-test-opentelekomcloud/run.yaml]: yes, will clean it

**Pull request merged**